lib/cli/parse.rs

```diff
@@ -1492,4 +1526,38 @@ mod tests {
             }
         }
     }
+    mod account_identifier{
+        use casper_types::account::AccountHash;
+        use crate::rpcs::v1_6_0::get_account::AccountIdentifier;
+        use super::*;
+        #[test]
+        pub fn should_parse_valid_account_hash() {
+            let account_hash = "account-hash-c029c14904b870e64c1d443d428c606740e82f341bea0f8542ca6494cef1383e";
+            let parsed = account_identifier(account_hash).unwrap();
+            let _expected = AccountHash::from_formatted_str(account_hash).unwrap();
```

Comment:
Nit: the leading underscore is normally only applied to unused variables. (Same comment on test below).